src/lib/CustomRulePathManager.ts

```diff
@@ -246,7 +246,7 @@ export class CustomRulePathManager implements RulePathManager {
 				throw SfdxError.create('@salesforce/sfdx-scanner', 'add', 'errors.invalidFilePath', [p]);
 			}
 			if (stats.isFile()) {
-				if (p.endsWith(".jar")) {
+				if (p.endsWith(".jar") || p.endsWith(".xml")) {
```

Comment:
Allow users to specify xml files. PMD CLI supports this, it seems like it would be good for the scanner to have parity.